### PR TITLE
support python 3.9

### LINKED ISSUE #171: Support Python 3.9
Related to this [comment](https://github.com/m3dev/gokart/pull/170#discussion_r566565954), gokart does not support python 3.9 currently.

### PR DESCRIPTION
**Pull request by hirosassa**
Support Python 3.9

closes #171

### CODE REVIEW

**Comment by vaaaaanquish** [comment]
cool!

File: pyproject.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ matplotlib = "*"
 
 [tool.poetry.dev-dependencies]
 tox = "*"
-moto = "==1.3.6"
+moto = "*"
```
Comment (hirosassa) [inline pyproject.toml:35]: This prevents supporting python 3.9

**Review by Hi-king — APPROVED** [review]
Cool! LGTM

**Comment by Hi-king** [comment]
@hirosassa thx for great contribution! I've merged this

**Comment by hirosassa** [comment]
@Hi-king Thanks!

**Pull request merged**